server/routes/initialAppointment/handlers/initialMeetingTime.ts

```diff
@@ -17,6 +21,7 @@ export default class InitialMeetingTimeRoutes {
       appointmentTimeType,
       releaseIsOnBankHolidayOrWeekend: licence.isEligibleForEarlyRelease,
       skipUrl: this.getNextPage(licence.id.toString(), licence.typeCode, req),
+      canSkip: this.userType === UserType.PROBATION,
```

Comment:
I think this could be based solely on whether the value is populated or not? 
We don't want to let them skip if they've previously provided a value (as it doesn't do anything and is confusing for the user)

Reply:
I can add that check, but I think it'll need to be in addition to the existing check as I don't believe we're allowing CAs to ever skip this option (eg if they go through the hard-stop licence creation, we don't want them to be able to skip)